### DISCUSSION TITLE
When can Mojo run on local

### DISCUSSION THREAD

**Discussion by owenisas** [modular/modular#64] [Q&A] [answered]
Running on playground seems to be a bit weird

**Comment by bnyu** [comment]
I guess when it be opensource

**Comment by iamtimdavis** [comment, marked as answer]
We are working hard to get done - you can see this in our [FAQ](https://docs.modular.com/mojo/faq.html#why-cant-i-run-mojo-locally). We'll announce it broadly when its available on specific device targets.

**Reply by TZDak** [reply]
I notice that the "#why-cant-i-run-mojo-locally" entry in the FAQ includes a link to the Discord channel. I would recommend adding a link there to this discussion forum also.

DAK
